src/components/WelcomeModal/CarouselCard.tsx

```diff
@@ -0,0 +1,63 @@
+import { useRef, useEffect } from "react";
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import { Body, Subtitle } from "@leafygreen-ui/typography";
+import { size } from "constants/tokens";
+import { OneOf } from "types/utils";
+import { ImageCardType, VideoCardType } from "./types";
+
+interface CarouselCardProps {
+  card: OneOf<ImageCardType, VideoCardType>;
+  visible: boolean;
+}
+
+const CarouselCard: React.VFC<CarouselCardProps> = ({ card, visible }) => {
```

Comment:
I wonder if it might be a good idea to disable text highlighting through css

https://user-images.githubusercontent.com/47064971/167454366-2da76564-fb82-43c8-9034-f7533fb40e95.mov